### PR TITLE
feat(tile): add support for active variant styling FE-5355

### LINKED ISSUE #5444: Add "Active" Variant to the tile component.
### Desired behaviour

Add a variant to the tile component for when it is shown within an Active state within the UI. The active state should use the same tokens as the tile select active variant. 

**Border token:**
colors.action.major.025

**Background token:**
colors.action.major.500

### Current behaviour

Currently the tile only has one state/variant with a white background colour.

### Suggested Solution

Add an active variant to allow us to achieve desired designs.

### CodeSandbox or Storybook URL

_No response_

### Anything else we should know?

_No response_

### Confidentiality

- [X] I confirm there is no confidential or commercially sensitive information included.

### PR DESCRIPTION
**Pull request by edleeks87**
fix #5444

### Proposed behaviour

Adds new `active` style `variant` option to `Tile` component.

image

### Current behaviour

No support for `active` `variant`

### Checklist

- [x] Commits follow our style guide
- [x] Related issues linked in commit messages if required
- [x] Screenshots are included in the PR if useful
- [x] Unit tests added or updated if required
- [x] Storybook added or updated if required
- [x] Typescript `d.ts` file added or updated if required

#### QA

- [ ] Tested in CodeSandbox/storybook
- [ ] Add new Cypress test coverage if required
- [ ] Carbon implementation matches Design System/designs
- [ ] UI Tests GitHub check reviewed if required

### Testing instructions

The following CodeSandbox is an example of the broken behaviour.
You can see the new behaviour by looking at the version in the comment by `codesandbox[bot]`.

### CODE REVIEW

File: src/components/tile/tile.d.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { SpaceProps } from "styled-system";
 
 export interface TileProps extends SpaceProps {
   /** Sets the theme of the tile - either 'tile' or 'transparent' */
```
Comment (nineteen88) [inline src/components/tile/tile.d.ts:5]: Update comment

File: src/components/tile/tile.component.js [hunk 2]
```diff
@@ -59,7 +59,7 @@ Tile.propTypes = {
   /** Styled system spacing props */
   ...propTypes.space,
   /** Sets the theme of the tile - either 'tile' or 'transparent' */
```
Comment (nineteen88) [inline src/components/tile/tile.component.js:61]: Update comment

**Review by grabkowski — COMMENTED** [review]
looks good, I'm happy to approve, when you resolve @nineteen88 comments.

**Review by nineteen88 — APPROVED** [review]

**Review by grabkowski — APPROVED** [review]

**Review by divyajindel — APPROVED** [review]

**Pull request merged**